modules/backend/widgets/Filter.php

```diff
@@ -470,20 +513,16 @@ public function addScopes(array $scopes)
                 $scopeObj->maxDate = '2099-12-31';
             }
 
+            /*
+             * Ensure dates options are set
+             */
+            if (!isset($config['minNumber'])) {
+                $scopeObj->minNumber = '0';
+                $scopeObj->maxNumber = '999999999';
```

Comment:
`$config['maxNumber']` could be set but that would be ignored if `$config['minNumber']` wasn't; the opposite is also true; i.e. minNumber could be set while maxNumber isn't. Those cases don't appear to be handled here.